app/recordtransfer/admin.py

```diff
@@ -77,6 +78,12 @@ def format_upload_size(obj: BaseUploadedFile) -> str:
     return get_human_readable_size(int(obj.file_upload.size), 1000, 2)
```

Comment:
I kept getting an error because one of the files I had uploaded didn't exist.

![Screenshot 2025-06-16 133217](https://github.com/user-attachments/assets/2fc939ae-bed6-426e-a082-80a9f79a3994)

![Screenshot 2025-06-16 133234](https://github.com/user-attachments/assets/5acd00e5-7e6d-4ea1-b9ce-84254bfd4e1b)

Can you change this to check that the file exists first?

```suggestion
    if not obj.file_upload or not obj.exists:
        return "N/A"
    return get_human_readable_size(int(obj.file_upload.size), 1000, 2)
```